ts/util/releaseFeature.ts

```diff
@@ -32,7 +30,7 @@ export async function checkIsFeatureReleased(featureName: string): Promise<boole
     const featureAlreadyReleased = await getIsFeatureReleased();
 
     // Is it time to release the feature?
-    if (Date.now() >= featureReleaseTimestamp) {
+    if (Date.now() >= FEATURE_RELEASE_TIMESTAMPS[`${featureName}`]) {
```

Comment:
I needed to do this because of moving the timestamp object to the constants file

```
export const FEATURE_RELEASE_TIMESTAMPS: Record<string, number> = {
  // TODO update to agreed value between platforms
  DISAPPEARING_MESSAGES_V2: 1706778000000, // unix 01/02/2024 09:00
  // NOTE for testing purposes only
  // DISAPPEARING_MESSAGES_V2: 1677488400000, // unix 27/02/2023 09:00
};
```